test/lyapunov.jl

```diff
@@ -51,3 +51,34 @@ if printinfo
 end
 return
 end
+
```

Comment:
This test is not enough.

Please add 2 more tests: that check if λ for very small magnetic fields is the same as λ without magnetic fields. Do this for 2 different billiards of your choice, doesn't matter.

(there has to be some small average over 10 particles or so)

Reply:
Talking about tests, can you add one more test that confirms what the function does when given a pinned particle?

```
bt = billiard_sinai(;setting="periodic")
p = MagneticParticle(0.1, 0.5, -π/2, 2.0)
Λ = lyapunovspectrum!(p, bt, 10000.0)
@test Λ[1] == 0
```